docs/guide/topics/error_handling.rst

```diff
@@ -0,0 +1,361 @@
+.. currentmodule:: discord
+
+
+==========================
+Errors and Error Handling
+==========================
+
+Handling Errors
+===============
+
+When an error ocurrs in a command or an event, it will, by default, show an error in the console to denote that the error 
+ocurred and has been silently ignored. In order to avoid this, an error handler is needed.
+
+In :ref:`Commands <discord_ext_commands>`
+-----------------------------------------
+
+All errors that are raised by a command derive from :exc:`~.ext.commands.CommandError`. If an error that occurs during a command does
+not inherit from :exc:`~.ext.commands.CommandError`, it will automatically get wrapped in a :exc:`~.ext.commands.CommandInvokeError`, 
+to retrieve/unwrap this error we need to access it's ``.original`` attribute.
+
+**Example**
+
+.. code-block:: python3
+
+    if isinstance(error, commands.CommandInvokeError):
+         error = error.original
+
+Global Error handlers
+~~~~~~~~~~~~~~~~~~~~~
+
+You can create global error handlers by creating :func:`.on_command_error` events, which work like any other event in 
+the :ref:`discord-api-events`, these are called for every exception that occurs in a command, before invoke hooks, command
+converters, etc.
+
+**Example**
+
+.. code-block:: python3
+    :emphasize-lines: 1, 2
+
+    @bot.listen()
+    async def on_command_error(ctx, error):
+        """ This is an example global error handler. """
+
+        # First we unwrap any error that gets wrapped in a CommandInvokeError
+        if isinstance(error, commands.CommandInvokeError):
+            error = error.original
+
+        # Then we continue with our actual logic:
+
+        # We use isinstance() to check which error ocurred:
+        if isinstance(error, commansd.CommandNotFound):
+            await ctx.send("I couldn't find that command, sorry!")
+
+        elif isinstance(error, commands.NotOwner):
+            await ctx.send("This command can only be used by my owner!")
+
+        else:  # Important!
+            print(f"Ignoring unhandled exception in command {ctx.command!r}")
+            traceback.print_exception(type(error), error, error.__traceback__)
+            
+
+.. note::
+    We don't want our global error handler to 'eat' our unhandled errors without displaying anything in the console, so we add
+    an :ref:`else <py:else>` statement at the end of our logic and handle all the remaining 'unhandled' errors.
+
+Local Error Handlers
+~~~~~~~~~~~~~~~~~~~~
+
+In :class:`Cogs <~ext.commands.Cog>`
+
+***************************************
+
+To handle errors in only a single :class:`~ext.commands.Cog` you override it's :meth:`~ext.commands.Cog.cog_command_error` method,
+which gets called when an error occurs inside of this Cog.
```

Comment:
```suggestion
which gets called when an error occurs inside of Command defined within this Cog.
```

Should probably be a bit more explicit here, since you can have listeners and other methods in the class that won't call this handler.